unattended_scripts/open-distro/tools/wazuh-passwords-tool.sh

```diff
@@ -168,35 +168,40 @@ User:
     fi
 
     if [ -n "${CHANGEALL}" ]; then
-        for i in "${!USERS[@]}"; do
-	    for j in "${!FILEUSERS[@]}"; do
-	        if [[ ${USERS[i]} == ${FILEUSERS[j]} ]]; then
-		    PASSWORDS[i]=${FILEPASSWORDS[j]}
-		fi
-	    done
+        for j in "${!FILEUSERS[@]}"; do
```

Comment:
These changes correspond to a fix related to https://github.com/wazuh/wazuh-packages/pull/937